rotkehlchen/tests/utils/blockchain.py

```diff
@@ -603,15 +603,16 @@ def set_web3_in_inquirer(ethereum_inquirer: 'EthereumInquirer', web3: Web3) -> N
     ethereum_inquirer.web3_mapping[node_name] = web3
 
 
-def setup_filter_active_evm_addresses_mock(
+def setup_evm_addresses_activity_mock(
         stack: 'ExitStack',
         chains_aggregator: 'ChainsAggregator',
-        contract_addresses: list[ChecksumEvmAddress],
+        eth_contract_addresses: list[ChecksumEvmAddress],
+        ethereum_addresses: list[ChecksumEvmAddress],
         avalanche_addresses: list[ChecksumEvmAddress],
         optimism_addresses: list[ChecksumEvmAddress],
 ) -> 'ExitStack':
     def mock_ethereum_get_code(account):
-        if account in contract_addresses:
+        if account in eth_contract_addresses:
```

Comment:
why renaming to `eth_contract_addresses`?

Reply:
To add more sense to the name. Because it is actually for eth contracts only, not for contracts on all chains